contracts/core/interfaces/ICoreAccounting.sol

```diff
@@ -0,0 +1,43 @@
+/*
+    Copyright 2018 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.4.24;
+
+
+/**
+ * @title ICoreIssuance
+ * @author Set Protocol
+ *
+ * The ICoreIssuance Contract defines all the functions exposed in the CoreIssuance
+ * extension.
+ */
+contract ICoreAccounting {
```

Comment:
It's a little weird to be using an abstract class here instead of `interface` (same for ICoreIssuance)...I think solidity throws warnings if methods in interfaces aren't `external` which wouldn't work here though but I wonder if we should be splitting up the naming of interfaces from abstract classes

Reply:
Yeah we can't use internal functions in libraries so we have to define it as a contract. Which unfortunately makes us have to be more careful with inheritance when the Core inherits the IssuanceOrder extension lest we overwrite the actual logic of the batchDepositInternal function. If we want to name it something else we can do that since it isn't technically an interface thought it is intended to act as such.